database/mysql.go

```diff
@@ -136,6 +138,9 @@ func (r *TestMySQLDB) Close() error {
 	return nil
 }
 
+var sharedMySQLConfig *DatabaseConfig
```

Comment:
Looking at `L167`, I think this type could be `*MySQLConfig`

Reply:
Good point!